ompi/mca/pml/ucx/pml_ucx.c

```diff
@@ -705,6 +706,7 @@ int mca_pml_ucx_isend_init(const void *buf, size_t count, ompi_datatype_t *datat
         OBJ_RETAIN(datatype);
     } else {
         req->datatype.datatype = mca_pml_ucx_get_datatype(datatype);
+        OMPI_DATATYPE_RETAIN(datatype);
```

Comment:
do we need to release refcount somewhere?

Reply:
Yes, will fix it.